docs/topics/collection-grouping.md

```diff
@@ -39,12 +39,37 @@ Namely, `Grouping` supports the following operations:
    This is the generic way to perform any operations on a `Grouping`. Use it to implement custom operations when fold or reduce are not enough.
 
 ```kotlin
-
 fun main() {
 //sampleStart
     val numbers = listOf("one", "two", "three", "four", "five", "six")
     println(numbers.groupingBy { it.first() }.eachCount())
 //sampleEnd
 }
 ```
-{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
\ No newline at end of file
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To iterate through the groups created by the `groupBy()` function, you can use the [`forEach`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each.html) function on the resulting `Map`.
+This allows you to access each key and the list of elements associated with that key.
+
+Let's look at an example where we group the strings by their first letter, convert it to uppercase,
+and then iterate through each group to print the key and its associated values:
+
+```kotlin
+fun main() {
+    val numbers = listOf("one", "two", "three", "four", "five")
```

Comment:
missing //sampleStart //sampleEnd comments

Reply:
good point -  I restructured it 👍